### DISCUSSION TITLE
ValidationError: No Op registered for LayerNormalization

### DISCUSSION THREAD

**Discussion by RobinRCM** [onnx/onnx#3998] [Q&A] [answered]
I used HuggingFace transformers' [convert_graph_to_onnx](https://github.com/huggingface/transformers/blob/master/src/transformers/convert_graph_to_onnx.py) script to convert a PyTorch model to the ONNX format.

When checking the model, I get:

```
File ".../python3.9/site-packages/onnx/checker.py", line 106, in check_model
C.check_model(protobuf_string)
onnx.onnx_cpp2py_export.checker.ValidationError: No Op registered for LayerNormalization with domain_version of 11

==> Context: Bad node spec for node. Name: LayerNormalization OpType: LayerNormalization
```

Does it mean that the LayerNormalization nodes of the graph aren't supported by ONNX?

Thank you

**Comment by WilliamTambellini** [comment, marked as answer]
I dont think there is an official op (yet ?) for LayerNorm despite the need/demand:
https://github.com/onnx/onnx/issues/2379

**Comment by jcwchen** [comment]
FYI: LayerNormalization will be included in upcoming ONNX 1.12 release: https://github.com/onnx/onnx/pull/4076.
